conda/recipes/libcugraph/meta.yaml

```
@@ -116,14 +159,22 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
     requirements:
       build:
         - cmake {{ cmake_version }}
       run:
         - {{ pin_subpackage('libcugraph_etl', exact=True) }}
         - {{ pin_subpackage('libcugraph', exact=True) }}
+        {% if cuda_major == "11" %}
         - cudatoolkit {{ cuda_spec }}
+        {% else %}
+        - cuda-cudart {{ cuda_spec }}
+        - cuda-cudart-dev {{ cuda_spec }}
+        - cuda-cudart-static {{ cuda_spec }}
+        {% endif %}
```

Remove `cuda_spec` as mentioned above. Also we can safely remove the `static` and `dev` packages from run requirements. In this case I also removed `cuda-cudart`, I think it's supplied automatically (at least we haven't included it in other CUDA 12 PRs?).
```suggestion
        - cudatoolkit
        {% endif %}
```